### PR TITLE
IntegrationScriptUnifier: use get_file to get the code file

### PR DESCRIPTION
**Pull request by ilaner**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-24892?filter=-1

## Description
* Added `return_content` argument to `get_file`, to return the content of the file
* Use the `get_file` in `IntegrationScriptUnifier` to avoid unicode errors.

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -812,6 +813,8 @@ def get_file(
 
     try:
         file_content = _read_file(file_path)
+        if return_content:
+            return file_content
```
Comment (dorschw) [inline demisto_sdk/commands/common/tools.py:817]:
```suggestion
        if return_content := _read_file(file_path):
            return file_content
```

**Review by dorschw — APPROVED** [review]
Please add a changelog entry before merging

File: demisto_sdk/commands/common/tools.py (hunk 1 above)
Comment (ilaner) [reply demisto_sdk/commands/common/tools.py]: This is not the correct behavior.
We want to read the file anyway, and return its content only of return_content is True

Comment (dorschw) [reply demisto_sdk/commands/common/tools.py]: You're right, it's getting late..

**Pull request merged**